moml/simulation/molecular_dynamics/force_field/plugins/gac_v1/build.py

```
@@ -0,0 +1,36 @@
+    idx_list = []
+
+    for i in range(nx):
+        for j in range(ny):
+            x = (i + 0.5 * (j % 2)) * a * np.sqrt(3)
+            y = j * a * 1.5
+            positions.append([x, y, slab_z / 2])
+            atom = topology.addAtom(f"C{i}_{j}", carbon, res)
+            idx_list.append(atom.index)
+
+    pdb_path = tmp_dir / "gac_slab.pdb"
+    PDBFile.writeFile(topology, positions, open(pdb_path, "w"))
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)